connect/html/index.html

```diff
@@ -29,9 +29,13 @@
             connectToken: accessToken,
             includeSandbox: true,
             onSuccess: (itemData) => {
+              // TODO: Implement logic for succesfull connection
+              // The following line is an example, it should be removed when implemented.
               console.log('Yay! Pluggy connect success!', itemData);
             },
             onError: (error) => {
+              // TODO: Implement logic for error on connection
+              // The following line is an example, it should be removed when implemented.
```

Comment:
we should add this comment in the rest of the examples as well?

Reply:
We could add it!